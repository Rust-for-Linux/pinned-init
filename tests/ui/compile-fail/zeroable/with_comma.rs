extern crate pin_init;
use pin_init::*;

#[derive(Zeroable)]
struct Foo {
    a: usize,
    b: usize,
}

fn main() {
    let _ = init!(Foo {
        a: 0,
        ..Zeroable::zeroed(),
    });
}
