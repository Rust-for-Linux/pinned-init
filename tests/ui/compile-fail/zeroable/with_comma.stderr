error: unexpected token, expected `}`
  --> tests/ui/compile-fail/zeroable/with_comma.rs:12:29
   |
12 |         ..Zeroable::zeroed(),
   |                             ^
